src/js/hca/components/HealthCareApp.jsx

```diff
@@ -129,23 +129,38 @@ class HealthCareApp extends React.Component {
     e.preventDefault();
     const veteran = this.props.data;
     const path = this.props.location.pathname;
+    let apiUrl = `${window.VetsGov.api.url}/api/hca/v1/application`;
+    let submissionPost = {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json'
+      },
+      timeout: 10000, // 10 seconds
+      body: veteranToApplication(veteran)
+    };
 
     window.dataLayer.push({ event: 'submit-button-clicked' });
     const formIsValid = validations.isValidForm(veteran);
 
-    if (formIsValid && veteran.privacyAgreementAccepted) {
-      this.props.onUpdateSubmissionStatus('submitPending');
-
-      // POST data to endpoint
-      fetch(`${window.VetsGov.api.url}/api/hca/v1/application`, {
+    if (__BUILDTYPE__ === 'development' || __BUILDTYPE__ === 'staging') {
+      apiUrl = `${window.VetsGov.api.url}/v0/health_care_applications`;
+      submissionPost = {
```

Comment:
i think this line should be `submissionPost.body = JSON.stringify({ form: submissionPost.body });`
i didn't see form in the fetch api docs, what the api wants in the request body is `{ form: [the form as a JSON string] }`, we want it to be JSON encoded twice because otherwise the middleware will change the casing of the form and that will break things